q2_fmt/citations.bib

```diff
@@ -0,0 +1,21 @@
+
+@article{aggarwala_precise_2021,
+	title = {Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes},
+	volume = {6},
+	issn = {2058-5276},
+	url = {https://www.nature.com/articles/s41564-021-00966-0},
+	doi = {10.1038/s41564-021-00966-0},
+	abstract = {Abstract
+            
+              Fecal microbiota transplantation ({FMT}) has been successfully applied to treat recurrent
+              Clostridium difficile
+              infection in humans, but a precise method to measure which bacterial strains stably engraft in recipients and evaluate their association with clinical outcomes is lacking. We assembled a collection of {\textgreater}1,000 different bacterial strains that were cultured from the fecal samples of 22 {FMT} donors and recipients. Using our strain collection combined with metagenomic sequencing data from the same samples, we developed a statistical approach named Strainer for the detection and tracking of bacterial strains from metagenomic sequencing data. We applied Strainer to evaluate a cohort of 13 {FMT} longitudinal clinical interventions and detected stable engraftment of 71\% of donor microbiota strains in recipients up to 5 years post-{FMT}. We found that 80\% of recipient gut bacterial strains pre-{FMT} were eliminated by {FMT} and that post-{FMT} the strains present persisted up to 5 years later, together with environmentally acquired strains. Quantification of donor bacterial strain engraftment in recipients independently explained (precision 100\%, recall 95\%) the clinical outcomes (relapse or success) after initial and repeat {FMT}. We report a compendium of bacterial species and strains that consistently engraft in recipients over time that could be used in defined live biotherapeutic products as an alternative to {FMT}. Our analytical framework and Strainer can be applied to systematically evaluate either {FMT} or defined live bacterial therapeutic studies by quantification of strain engraftment in recipients.},
```

Comment:
```suggestion
	abstract = {Fecal microbiota transplantation ({FMT}) has been successfully applied to treat recurrent Clostridium difficile infection in humans, but a precise method to measure which bacterial strains stably engraft in recipients and evaluate their association with clinical outcomes is lacking. We assembled a collection of 1,000 different bacterial strains that were cultured from the fecal samples of 22 {FMT} donors and recipients. Using our strain collection combined with metagenomic sequencing data from the same samples, we developed a statistical approach named Strainer for the detection and tracking of bacterial strains from metagenomic sequencing data. We applied Strainer to evaluate a cohort of 13 {FMT} longitudinal clinical interventions and detected stable engraftment of 71\% of donor microbiota strains in recipients up to 5 years post-{FMT}. We found that 80\% of recipient gut bacterial strains pre-{FMT} were eliminated by {FMT} and that post-{FMT} the strains present persisted up to 5 years later, together with environmentally acquired strains. Quantification of donor bacterial strain engraftment in recipients independently explained (precision 100\%, recall 95\%) the clinical outcomes (relapse or success) after initial and repeat {FMT}. We report a compendium of bacterial species and strains that consistently engraft in recipients over time that could be used in defined live biotherapeutic products as an alternative to {FMT}. Our analytical framework and Strainer can be applied to systematically evaluate either {FMT} or defined live bacterial therapeutic studies by quantification of strain engraftment in recipients.},
```

Reply:
I combined a few changes into one suggestion, which makes the diff hard to read - but my suggestion includes removing the extra spacing on lines 8-11, removing 'Abstract' as the first word in the paragraph, and removing the `{\textgreater}` closing tag, because that just shows up as plain text on the command line.